robot/Cumulus/tests/browser/automated_soft_credits/asc_primary_and_affiliation.robot

```diff
@@ -2,43 +2,70 @@
 
 Resource        robot/Cumulus/resources/NPSP.robot
 Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/OpportunityContactRolePageObject.py
 ...             robot/Cumulus/resources/NPSPSettingsPageObject.py
-Suite Setup     Open Test Browser
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
 
+***Keywords***
+Setup Test Data
+    Setupdata           account      None    None    ${account_fields}
```

Comment:
We can replace Nones, by specifying something like
Setupdata   account   account_data=${account_fields}     
and say, if you want to create an opportunity associated with that account within the same step
Setupdata   account   account_data=${account_fields}    opportunity_data=${opportunity_fields}

Reply:
I tried Setupdata account account_data=${account_fields} and it didn't work, was giving me errors so had to go with this as last option.

Reply:
And opportunity cannot be created together until the stuff is created so cant do it together. Also was having issue with opp created for acct but noticed u added code in ur PR for that so didn't add in mine.